assets/react/v3/entries/course-builder/components/modals/LessonModal.tsx

```diff
@@ -130,12 +134,21 @@ const LessonModal = ({
           prerequisites: lessonDetails.content_drip_settings?.course_prerequisites || [],
         },
       });
+
+      timeoutId = setTimeout(() => {
+        form.setFocus('title');
+      }, 0);
```

Comment:
What is the purpose of this?

Reply:
This will make sure, form.setFocus will run after the whole form has been rendered. Without this the form.setFocus trigger does not show any change in the focus state on the front end. I tried triggering setFocus after resetting the whole form., it doesn't work.